hops/util.py

```diff
@@ -204,6 +212,10 @@ def assumed_role():
     get_secret_value_response = client.get_secret_value(SecretId=secret_name)
     return json.loads(get_secret_value_response['SecretString'])[secret_key]
 
+def query_paramter_store(project_name, secret_key):
```

Comment:
Is it a public method? If so add docstring. If not, prepend "_" to the methodname.

Reply:
Also, there is a typo in the method name: `paramter`.

Reply:
Shouldn't be public anymore. I fixed that and the typo.